esphome/components/ina2xx_spi/sensor.py

```diff
@@ -0,0 +1,27 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import ina2xx_base, spi
+from esphome.const import CONF_ID
+
+CODEOWNERS = ["@latonita"]
+AUTO_LOAD = ["ina2xx_base"]
```

Comment:
Small nitpick: alphabetical order =)
```suggestion
AUTO_LOAD = ["ina2xx_base"]
CODEOWNERS = ["@latonita"]
```